### DISCUSSION TITLE
请问是否考虑添加av1编码下载选项

### DISCUSSION THREAD

**Discussion by netphils** [the1812/Bilibili-Evolved#3089] [功能建议 / Ideas]
注意到了b站有些视频添加了AV1编码方式。请问是否考虑在下载格式里添加dash(AV01)？
![image](https://user-images.githubusercontent.com/100473503/156888799-fa086911-2dc0-407a-9a64-b61938b63fc0.png)

**Comment by Geeyun-JY3** [comment]
请善用搜索：https://github.com/the1812/Bilibili-Evolved/discussions/3052#discussioncomment-2234995
重复：#2941、#3052
